spec/default_facts.yml

```diff
@@ -2,7 +2,8 @@
 #
 # Facts specified here will override the values provided by rspec-puppet-facts.
 ---
-ipaddress: "172.16.254.254"
-ipaddress6: "FE80:0000:0000:0000:AAAA:AAAA:AAAA"
+networking:
+    ip: "172.16.254.254"
```

Comment:
Why 4 indents, isn't 2 more common?

Reply:
Doh!
Yes, my mistake.
Will update the templates so this is corrected in the next update.